src/NHibernate.Test/CacheTest/QueryCacheFixture.cs

```diff
@@ -60,5 +60,19 @@ public void QueryCacheWithNullParameters()
 				s.Flush();
 			}
 		}
+
+		[Test]
+		public void QueryCacheWithScalarFunctionThatReturnsInt()
+		{
+			using (ISession s = OpenSession())
+			{
+				int result = s
+					.CreateSQLQuery("SELECT 200012")
+					.SetCacheable(true)
+					.UniqueResult<int>();
+
+				Assert.IsNotNull(result);
+			}
+		}
 	}
-}
```

Comment:
Avoid, if possible

Reply:
It's .editorconfig setting, so it's fine.